### ISSUE TITLE
Gallery-slider component uses file path as alt-text

### ISSUE THREAD

**Issue by TeaDrinkingProgrammer** [gethugothemes/hugo-modules#52]
The Gallery-slider component uses the path of the file as alt-text. This is probably worse than having no alt-text at all for people using a screenreader. Using the filename itself would be slightly better (which is probably the intention of the component). It is also probably a good idea to think about a better way, like using the comment field in the EXIF data as alt-text.

**Comment by tfsomrat** [comment]
Hey @TeaDrinkingProgrammer, thanks for your concern, I have updated the module. now it's use only the file name.
